app/app.go

```diff
@@ -142,6 +145,12 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return err
 	}
 
+	lockHash, err := lock.HashTreeRoot()
+	if err != nil {
+		return err
+	}
+	lockHashHex := hex.EncodeToString(lockHash[:7])
```

Comment:
this means `lockHashHex` is 14 chars long. is this okay or can we lower it?

Reply:
will lower thanks!

Reply:
```suggestion
	lockHashHex := hex.EncodeToString(lockHash[:])[:7]
```